src/main/java/com/karankumar/bookproject/ui/goal/ReadingGoalView.java

```diff
@@ -78,6 +78,11 @@
      */
     Span booksToRead;
 
+    /**
```

Comment:
I know there's a javadoc comment for the other instance variables, but I was going to refactor that. Could you rename this variable to something descriptive such that including the javadoc comment would be redundant?